src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -175,14 +177,47 @@ public void assertNotBlank(AssertionInfo info, CharSequence actual) {
   }
 
   private boolean isBlank(CharSequence actual) {
-    if (actual == null || actual.length() == 0) return false;
+    return isNullOrEmpty(actual) || notNullSeqContainsOnlyWhitespaces(actual);
+  }
+
+  private boolean containsOnlyWhitespace(CharSequence actual) {
+    return !isNullOrEmpty(actual) && notNullSeqContainsOnlyWhitespaces(actual);
+  }
+
+  private boolean isNullOrEmpty(CharSequence actual) {
+    return (actual == null || actual.length() == 0);
+  }
+
+  private boolean notNullSeqContainsOnlyWhitespaces(CharSequence actual) {
     for (int i = 0; i < actual.length(); i++) {
-      if (!Whitespace.isWhitespace(actual.charAt(i))) return false;
+      if (!Character.isWhitespace(actual.charAt(i))) return false;
```

Comment:
minor static import `isWhitespace` for readability

Reply:
Done